app/src/main/res/layout/fragment_browser_tab.xml

```diff
@@ -66,6 +66,12 @@
         android:layout_height="match_parent"
         android:visibility="gone" />
 
+    <com.duckduckgo.app.browser.webview.MaliciousSiteBlockedWarningLayout
```

Comment:
Nit: I wouldn't change it now, but eventually, we should try to find a way to unify all different error types